src/Command/DispatchCommand.php

```diff
@@ -319,32 +319,53 @@ private function updateBranchesProtection(Package $package, array $projectConfig
             return;
         }
 
-        $protectionConfig = [
-            'required_status_checks' => [
-                'strict' => false,
-                'contexts' => [
-                    'continuous-integration/travis-ci',
-                ],
-            ],
-            'required_pull_request_reviews' => [
-                'dismissal_restrictions' => [
-                    'users' => [],
-                    'teams' => [],
-                ],
-                'dismiss_stale_reviews' => true,
-                'require_code_owner_reviews' => true,
-            ],
-            'restrictions' => null,
-            'enforce_admins' => false,
-        ];
         foreach ($branches as $branch) {
             $this->githubClient->repo()->protection()
-                ->update(static::GITHUB_GROUP, $repositoryName, $branch, $protectionConfig)
-            ;
+                ->update(static::GITHUB_GROUP, $repositoryName, $branch, [
```

Comment:
each branch gets its own protection: for example, on 3.x we will have required php 7.2 build, but on master we will not, because php 7.2 was dropped